sdk/highlight-node/src/hooks.ts

```diff
@@ -33,18 +36,22 @@ interface ConsolePayload {
 	level: keyof typeof ConsoleLevels
 	message: string
 	stack: object
+	attributes?: { [key: string]: any }
 }
 
 type ConsoleFn = (...data: any) => void
 
 let consoleHooked = false
 
 export function safeStringify(obj: any): string {
-	function replacer(input: any): any {
+	function replacer(input: any, depth?: number): any {
+		if ((depth ?? 0) > MAX_RECURSION) {
```

Comment:
Curious the impact if something where to be over the `MAX_RECURSION` value? The value seems low to the minimum browser results.

Reply:
I think it's unlikely an object someone wants to serialize intentionally will be deeper than 128 children, while keeping the value low means it's faster to detect a cycle in the object.